coresdk/src/coresdk/camera.cpp

```diff
@@ -168,12 +168,60 @@ namespace splashkit_lib
         move_camera_by(offset.x, offset.y);
     }
 
-    void center_camera_on(sprite s, double offset_x, double offset_y)
-    {
-        point_2d center = sprite_position(s);
+    // void center_camera_on(sprite s, double offset_x, double offset_y)
```

Comment:
These comments should not remain in the code. Only include comments that add value.